src/schema/v2/sale/__tests__/index.test.ts

```diff
@@ -836,6 +836,42 @@ describe("Sale type", () => {
       })
       expect(response.sale.formattedStartDateTime).toContain("Ends")
     })
+
+    it("returns date range without the word start interval is true before the sale has opened", async () => {
+      const response = await execute(query, {
+        start_at: moment().add(2, "hours"),
+        cascading_end_time_interval: 120,
+      })
+      expect(response.sale.formattedStartDateTime).not.toContain("Starts")
+    })
+
+    it("returns date range without the words In progress when cascading interval is true while the sale is running", async () => {
```

Comment:
Similar to above, I'm not sure I understand the point of this spec. Why not assert the various cases output the exact expected string? (rather than use `moment()` to return the current time, can use hard-coded dates to make stricter assertions around the entire expected string).